archtec/scripts/common.lua

```diff
@@ -27,6 +27,7 @@ function archtec.revoke_priv(name, priv)
 end
 
 archtec.silent_leave = {}
+
```

Comment:
Since there are two methods that reuse the same table, I thought it would be imperative to separate it with a line, just like another function was.